NineChronicles.Standalone.Tests/TableSheetsImporter.cs

```diff
@@ -0,0 +1,27 @@
+namespace NineChronicles.Standalone.Tests
+{
+    using System.Collections.Generic;
+    using System.IO;
+
+    public static class TableSheetsImporter
+    {
+        public static Dictionary<string, string> ImportSheets()
+        {
+            var sheets = new Dictionary<string, string>();
+            var dir = Path.Combine("Data", "TableCSV");
+            var files = Directory.GetFiles(dir, "*.csv", SearchOption.AllDirectories);
+            foreach (var filePath in files)
+            {
+                var fileName = Path.GetFileName(filePath);
+                if (fileName.EndsWith(".csv"))
```

Comment:
It seems to get files only matched to the pattern, `*.csv` by the line

```csharp
            var files = Directory.GetFiles(dir, "*.csv", SearchOption.AllDirectories);
```

Should it be checked one more time? then I wonder the reason.

Reply:
In fact, this `TableSheetsImporter` is copy from `Lib9c.Tests`. IDK why double-checked file extension. 🤔  I'll test it is ok to remove the `if statement`.